# Message that combines range data with pose
sensor_msgs/Imu imu
sensor_msgs/Range range
